pkgs/data/themes/sweet-nova/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, gitUpdater
+,
+}:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion

stdenvNoCC.mkDerivation {
```

Reply:
You removed the `rec`, without it I wasn't able to set sourceRoot though. Is it fine the way I updated it?

Reply:
It is OK. `rec` should be avoided.